### ISSUE TITLE
Zwift not recognise device

### ISSUE THREAD

**Issue by Sierrabravo1916** [cagnulein/qdomyos-zwift#2787]
Hi

as per emails, the beta doesn’t keep connection or if it does then Zwift will not recognise the device. Or show power/ speed etc
[debug-Sat_Nov_16_13_22_20_2024.log](https://github.com/user-attachments/files/17784858/debug-Sat_Nov_16_13_22_20_2024.log)
[debug-Sat_Nov_16_13_20_02_2024.log](https://github.com/user-attachments/files/17784859/debug-Sat_Nov_16_13_20_02_2024.log)

Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Append a debug log**

 Follow this guide https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
  
  
**Additional context**

**Comment by cagnulein** [comment]
hi @Sierrabravo1916 I deeply check the log thanks. The connection with the trainer is strong and it sends the metrics correctly however it changes them only once for quickly period. Are you sure you restarted the trainer? without qz is working on zwift now? because the counter of the spinning wheel, it is counting badly. You can also try to remove the app and start from 0 without the beta to check if it's working, but i'm 99% sure that it's not a beta issue or a qz issue. I hope to be wrong
Let me know!

**Comment by cagnulein** [comment]
just to be sure: in this session were you riding all the time? or just some spin sometimes? maybe that's the issue
let me know!

**Comment by Sierrabravo1916** [comment]
Hi

I was trying to do it riding for a period but it would t connect. I just
took my turbo apart and it’s fine.

**Comment by cagnulein** [comment]
sorry what do you mean your turbo?
in the log i see the connection to Tacx Neo correctly in qz and i can see the metrics. the problem is that the metrics were there for only quick periods, like you were spinning for 1 second and then stop.

Maybe i'm missing something in translation, I'm italian.
Thanks

**Comment by Sierrabravo1916** [comment]
Hiya

Thanks. I managed to get it connected on the app fine this time after
restarting everything.
However Zwift on the laptop found QZ app but not the wahoo kickr option for
second connection.

So it worked this time but not with the virtual gears which is what I
wanted to achieve.

I went back through the app settings and set zwift gears, reset but no joy

I’m not sure what is wrong. I appreciate your efforts to resolve this

Cheers


Sean

**Comment by cagnulein** [comment]
Ok that's another issue, its about windows firewall usually. Is it enabled?
Try to disable it.
Otherwise you can use zwift play directly on qz, or try zwift on another
device


Let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 16 nov 2024 alle 15:53 Sierrabravo1916 <
***@***.***> ha scritto:

**Comment by Sierrabravo1916** [comment]
I will check thank you, the laptop did find it before but then installed
some updates as I reset it also when trying get it all to work

I will try using Zwift play in qz app- is that by changing the setting in
the accessories menu?

Many thanks


Sean

**Comment by cagnulein** [comment]
Yes just enable the zwift play under accessories settings in qz
Let me know!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 16 nov 2024 alle 16:00 Sierrabravo1916 <
***@***.***> ha scritto:

**Comment by Sierrabravo1916** [comment]
Hi

I got it to work after four hours. Hopefully will keep going now!!

Cheers


Sean

**Comment by cagnulein** [comment]
Glad to hear it!
Let me know! Usually shouldn't be so hard :)

Il giorno sab 16 nov 2024 alle 16:09 Sierrabravo1916 <
***@***.***> ha scritto:
